### DISCUSSION TITLE
Graphql 22 Support

### DISCUSSION THREAD

**Discussion by cmuchinsky** [graphql-java-kickstart/graphql-java-servlet#572] [Q&A] [answered]
Now that GraphQL Java 22 has been out there for a few months, any plans to release GraphQL 22 compatible versions of the GraphQL Java Kickstart components?

**Comment by federicorispo** [comment, marked as answer]
@cmuchinsky In a couple of weeks we are releasing a new version of graphql-java-servlet with the new version of graphql-java! Stay tuned and thanks for your question

**Reply by cmuchinsky** [reply]
Thank you! Will there be a corresponding graphql-java-tools version released at the same time?

**Reply by cmuchinsky** [reply]
Any updates on when this might be released?

**Reply by federicorispo** [reply]
The graphql-java-servlet version [16.0.0](https://github.com/graphql-java-kickstart/graphql-java-servlet/releases/tag/v16.0.0extension) is released :rocket:

**Reply by cmuchinsky** [reply]
Very much appreciated @federicorispo, I have verified that compilation of my project is now working with GraphQL 22. Unfortunately I'm unable to fully vet this at runtime as I also depend on `graphql-java-tools` which hasn't been upgraded to support GraphQL 22 yet.

**Reply by federicorispo** [reply]
@oryan-block Can you please bump the `graphql-java` dependency in the `graphql-java-tools` project? If you don't have time, I can handle this bump for you

**Comment by thaparraj** [comment]
Can you please sepcify what libraries should I include for graphQL 22.0? I am facing an issue - NoClassDefFoundError for DataLoaderDispatcherInstrumentationOptions

**Reply by federicorispo** [reply]
@thaparraj Can you try with the newer version of the graphql-java-servlet (16.0.0)?
